### ISSUE TITLE
Map style changing issue

### ISSUE THREAD

**Issue by bourgeoismaxime** [mapbox/mapbox-unity-sdk#1106]
Hello,
I updated to Mapbox 2.0 (on Unity 2018.2.8f1) from the previous version and wanted to change the style of my map. So I created a new style and changed the setting in the map in the editor.
However it seems like there is some sort of conflict or some cache that I cannot delete as it is visible on this screenshot :
![image](https://user-images.githubusercontent.com/11771615/47455156-1e203800-d79f-11e8-98b7-1ec0a7e11abb.png)
The yellow/green zone is the old style being still rendered. I tried all the ways I found in order to clear the cache but the old style zone still remains.

Maxime

**Comment by brnkhy** [comment]
@bourgeoismaxime I believe these are Mapbox Studio styles right? If you changed and saved over the existing style (instead of creating a new one), as you guessed previous images will be cached and probably will be reused as they are cached by the very same style id. You can clear the caches from top menu `Mapbox -> Clear File Caches`. As the name suggest this will clear all saved images and force system to download all again.
If they are two separate styles (with separete keys I mean) and you're having this on switch, that's probably whole another bug. But let's verify that first possibility first ☝️

**Comment by bourgeoismaxime** [comment]
@brnkhy Thanks for the quick answer.
These are two different Mapbox styles with different style URLs. I already tried the Clear File Caches option, to set the caches sizes to 0 and also to delete the cache.db file in the persistant data folder. Even though all the map tiles was reloaded (the tiles appear black with the grid at first), I still got that style conflict. I opened the project this morning, it was fixed but there is very likely a bug because I couldn't get rid of the conflict yesterday.
